test/test_in_docker.sh

```diff
@@ -20,7 +20,7 @@ function buildTestRunnerImage() {
 function deployConjur() {
   pushd ..
     git clone --single-branch \
-      --branch master \
+      --branch deploy-oss-by-default \
```

Comment:
is this a branch that is already pushed to the k8s-conjur-deploy master?

Reply:
it is a tagged commit in `master`, not a branch.